pages/common/go.md

```diff
@@ -0,0 +1,23 @@
+# go
+
+> Tool for managing go source code.
+
+- Download and install a package, specified by its import path:
+
+`go get {{package_path}}`
+
+- Compile and run a source file (it has to contain a "main" package):
+
+`go run {{file}}.go`
+
+- Compile the package present in the current directory:
+
+`go build`
+
+- Execute all test cases of the current package (files have to end with "_test.go"):
```

Comment:
hm. github is trating the underscore as the start of italicized text, which then runs to the end of the document. How about using `` instead of ""?